packages/theme-chalk/src/form-item.scss

```
@@ -55,9 +55,14 @@ $form-item-label-top-margin-bottom: map.merge(
 );
 
 @include b(form-item) {
+  @include set-css-var-value(
+    ('form-item', 'margin-bottom'),
+    map.get($form-item-margin-bottom, 'default')
+  );
+
   display: flex;
   --font-size: #{map.get($input-font-size, 'default')};
-  margin-bottom: #{map.get($form-item-margin-bottom, 'default')};
+  margin-bottom: getCssVar('form-item', 'margin-bottom');
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)